corehq/apps/app_manager/suite_xml/post_process/remote_requests.py

```diff
@@ -200,7 +200,7 @@ def _remote_request_query_datums(self):
         additional_types = set(self.module.search_config.additional_case_types) - {self.module.case_type}
         datums = [
             QueryData(key='case_type', ref=f"'{case_type}'")
-            for case_type in [self.module.case_type] + list(additional_types)
+            for case_type in [self.module.case_type] + sorted(additional_types)
```

Comment:
Is this a concept that would make sense to abstract to ensure it's consistent?  (feel free to say no)

Reply:
I did consider that but there are subtle difference in how the data is passed in that would make it less useful. Though perhaps just having a function would make it clear that they should be consistent. I'll do a follow up PR.

Reply:
https://github.com/dimagi/commcare-hq/pull/30829